be/test/storage/rowset/segment_v2/column_reader_writer_test.cpp

```diff
@@ -198,6 +198,36 @@ class ColumnReaderWriterTest : public testing::Test {
                         }
                     }
                 }
+
+                {
```

Comment:
better to add cases with more than one column and different column types, which can validate the logic of read by range in page decoder